src/CLIFramework/IO/StreamWriter.php

```diff
@@ -11,6 +11,10 @@ class StreamWriter implements Writer
 
     public function __construct($stream)
     {
+        if (!is_resource($stream)) {
+            throw new \RuntimeException("invalid stream");
+        }
```

Comment:
yes, the use of root level class is preferred to be added on the top for consistency coding style.